### PR TITLE
Make error tab columns customizable

### PR DESCRIPTION
**Pull request by kahowell**

### CODE REVIEW

**Comment by alechenninger** [comment]
FYI I'm going to open a PR to add some very basic karma testing infrastructure, it would be good to start writing unit tests with all new frontend changes

**Comment by derek63** [comment]
@alechenninger I just merged your karma PR, should we add some tests for this one before merging?

**Comment by alechenninger** [comment]
@derek63 Thank you, and yes. I think I'm going to add tests for Kevin's changes once he is finished. He still has to modify the results section so it is resizable, but I guess the plugin that is being used for the resizable parent container does not emit events when it is resized, so there's not a clear way to know when to resize the children in the container.

@kahowell Perhaps you can make this happen via CSS rules?

File: app/src/main/webapp/partials/errors.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div id="errorpane">
-    <div ng-controller="ErrorsSearchCtrl">
+    <div ng-controller="ErrorsSearchCtrl" class="col-xs-12">
         <div class="panel panel-default">
```
Comment (alechenninger) [inline app/src/main/webapp/partials/errors.html]: Try (and then see below comment):

```html
    <div ng-controller="ErrorsSearchCtrl" class="row">
        <div class="col-xs-12 panel panel-default">
```

File: app/src/main/webapp/partials/errors.html [hunk 2]
```diff
@@ -26,43 +26,52 @@
...
-                        <tr ng-repeat="header in message.headers">
-                            <td>{{header.name}}</td>
-                            <td>{{header.type}}</td>
-                            <td>{{header.value}}</td>
-                        </tr>
-                    </tbody>
-                </table>
-            </tab>
-        </tabset>
+    <div class="col-xs-12" id="errorDisplayPane">
```
Comment (alechenninger) [inline app/src/main/webapp/partials/errors.html]: If you wrap this in another div with a `"row"` class, and also make the above comment's modifications, the two row's will line up cleaner.

**Comment by alechenninger** [comment]
I opened a PR against your branch with above fix and merging in master: https://github.com/esbtools/esb-message-admin/pull/63

**Comment by alechenninger** [comment]
Screwed up that PR heh, sec

**Comment by alechenninger** [comment]
Okay, here is a PR against your PR :) https://github.com/kahowell/esb-message-admin/pull/1

**Comment by alechenninger** [comment]
Thank you sir! Looks good! Working on a few tests now.

**Pull request merged**